.github/workflows/android.yml

```diff
@@ -0,0 +1,26 @@
+name: Android CI
+
+on:
+  push:
+    branches: [ master ]
```

Comment:
Should we include the other branches here too? Or will this action be included in the other long-living branches as well?